### PR TITLE
2.11.0-M3

### PR DESCRIPTION
**Pull request by retronym**
Review by @adriaanm

### CODE REVIEW

File: hand-written.md [hunk 1]
```diff
@@ -1,85 +1,50 @@
...
+      were shipped. We have now removed the old implementations.
+    * scala-actors is now deprecated; we advise users to follow the steps in the [Actors Migration Guide](http://docs.scala-lang.org/overviews/core/actors-migration-guide.html) to port to Akka Actors, which have been included in
+      the distribution since 2.10.0.
+    * Search and destroy mission for ~5000 chunks of dead code. (#1648)[https://github.com/scala/scala/pull/1648/files]
+* Language
+    * Case classes with > 22 parameters are now supported [SI-7296](https://issues.scala-lang.org/browse/SI-7296)
+    * Infer bounds of existential types [SI-1786](https://issues.scala-lang.org/browse/SI-1786)
+* REPL
+    * The bytecode decompiler command, :javap, now works with Java 7 [SI-4936](https://issues.scala-lang.org/browse/SI-4936) and has sprouted new options [SI-6894](https://issues.scala-lang.org/browse/SI-6894) (Thanks, Andrew Marki!)
+    * Added command :kind to help to tell ground types from type constructors. [#2340](https://github.com/scala/scala/pull/2340) (Thanks, George Leontiev and Eugene Yokata!)
```
Comment (eed3si9n) [inline hand-written.md]: Typo on my last name: Yokota.

File: hand-written.md [hunk 2]
```diff
@@ -1,85 +1,50 @@
...
+      and the REPL. In this release, all of these modules are still packaged in scala-compiler.jar.
+      We plan to ship them in separate JARs as of the next milestone. Similar work is planned in the standard
+      library.
+* Slimming down
+    * The experimental .NET backend has been removed from the compiler.
+    * In Scala 2.10.0, new implementations of the Pattern Matcher and the Bytecode Emitter
+      were shipped. We have now removed the old implementations.
+    * scala-actors is now deprecated; we advise users to follow the steps in the [Actors Migration Guide](http://docs.scala-lang.org/overviews/core/actors-migration-guide.html) to port to Akka Actors, which have been included in
+      the distribution since 2.10.0.
+    * Search and destroy mission for ~5000 chunks of dead code. (#1648)[https://github.com/scala/scala/pull/1648/files]
```
Comment (adriaanm) [inline hand-written.md]: Swapped braces: [#1648](https://github.com/scala/scala/pull/1648/files)

File: hand-written.md (hunk 1 above)
Comment (adriaanm) [inline hand-written.md]: I'd also consider turning people's names into links to their github profile.

File: src/main/scala/MakeReleaseNotes.scala [hunk 3]
```diff
@@ -41,6 +41,7 @@ object MakeReleaseNotes {
 
     s"""<html>
       <head>
+        <meta charset="UTF-8">
```
Comment (adriaanm) [inline src/main/scala/MakeReleaseNotes.scala:44]: :heart:  thanks for fixing the weird letters! :neckbeard:

File: hand-written.md (hunk 1 above)
Comment (retronym) [inline hand-written.md]: What, and risk more `[]` `()` permutations!?

Just kidding, that's a good idea. Thanks for the fine toothcomb review!

**Comment by retronym** [comment]
@adriaanm Updated

/cc @JamesIry

**Comment by retronym** [comment]
@adriaanm I don't have permissions to merge this.

**Comment by adriaanm** [comment]
> @adriaanm I don't have permissions to merge this.

@retronym that should be fixed now

**Pull request merged**